pkg/apis/wksprovider/machine/config/kubeadm/kubeadm.go

```diff
@@ -36,11 +40,16 @@ func NewClusterConfiguration(params ClusterConfigurationParams) *kubeadmapi.Clus
 	SANs = append(SANs, params.ExternalLoadBalancer)
 	SANs = append(SANs, params.AdditionalSANs...)
 
+	fmt.Println("Passed service CIDR block: ", params.ServiceCIDRBlock)
```

Comment:
We want to keep our output to the end user as clean as possible.  Please either remove this or convert to a log statement.  I suspect this was just an opps left in from debugging.

Reply:
D'oh